notification-hubs-sdk-e2etestapp/src/main/AndroidManifest.xml

```diff
@@ -6,10 +6,6 @@
 
     <uses-permission android:name="android.permission.INTERNET"></uses-permission>
 
-    <uses-sdk
```

Comment:
Android Studio alerted me to this error. Basically, this information shouldn't be present in the `AndroidManifest.xml` files, because it's already present in the `build.gradle` files.